pysat/tests/test_orbits.py

```diff
@@ -174,55 +197,57 @@ def test_all_single_orbit_calls_in_day(self):
             self.etime += relativedelta(seconds=((self.inc_min * 60) - 1))
             assert inst.index[-1] == self.etime
             assert self.testInst.index[-1] == self.etime
+        return
 
     def test_orbit_next_call_no_loaded_data(self):
-        """ Test orbit next call without loading data
-        """
+        """Test orbit next call without loading data."""
         self.testInst.orbits.next()
         assert (self.testInst.index[0] == dt.datetime(2008, 1, 1))
         assert (self.testInst.index[-1] == dt.datetime(2008, 1, 1, 0, 38, 59))
```

Comment:
Should these times be related back to one of the test class attributes?

Reply:
Here and elsewhere.

Reply:
I think this is all hard wired.  @rstoneback?

Reply:
This test and the next one are confusing me.  I get that if no data is loaded it loops around to the last file.  But why is the orbit end 38 minutes here and 11 minutes for the other test?  If we change the test date down the line, are these numbers going to be the same, or will it find different break points?

Reply:
Did you create an issue for these fixed date tests?

Reply:
The test date is set on when MLT rolls-over from 24 back to 0, plot attached. 
![orbits_mlt](https://user-images.githubusercontent.com/3595641/128202288-b1832a8f-f17a-42dd-aa7a-3c5ff0e226b9.png)

No, if the test date is changed it won't be the same. The correct times can be calculated using the known period and root_date of the 'met' signal.

Reply:
The start time check is driven by the fact that it is the first file loaded (last file in following test). The file dates in self.testInst.files.files.index could be used for a more generalized approach.